airflow/www/views.py

```diff
@@ -1790,10 +1790,10 @@ def variables(self, form):
     @wwwutils.action_logging
     def varimport(self):
         try:
-            out = str(request.files['file'].read())
+            out = request.files['file'].read().decode("utf-8")
```

Comment:
Since it's a file, could we use [json.load](https://docs.python.org/2/library/json.html#json.load) instead of [json.loads](https://docs.python.org/2/library/json.html#json.loads) and deal with encoding directly?

```python
d = json.load(request.files['file'])
```

Reply:
My preference would also be `json.load`.